extension-workspace-reference.md

```diff
@@ -0,0 +1,121 @@
+# workspace/xreference extension to LSP
```

Comment:
Oh, sorry missed that.
I would argue that `workspace/symbol` includes a non-optional query, so as a user, you search for one single symbol. `textDocument/documentSymbol` is also singular, but `textDocument/references` is plural, because the user always expects multiple references back.